lib/pbench/agent/tool_meister.py

```diff
@@ -285,6 +285,133 @@ def wait(self):
         self.start_process = None
 
 
+class PcpTransTool(Tool):
+    """PcpTransTool - A more traditional alternative to the pcp persistent tool that
+                      allows one to run both the pmlogger and pmcd locally on each
+                      registered node. Additionally, this tool starts, stops, and
+                      sends data alongside other transient tools, rather than always
+                      running in the background.
+    """
+
+    # Default path to the "pmcd" executable.
+    _pmcd_path_def = "/usr/libexec/pcp/bin/pmcd"
+
+    # Default path to the "pmlogger" executable.
+    _pmlogger_path_def = "/usr/bin/pmlogger"
+
+    def __init__(self, name, tool_opts, logger=None, **kwargs):
+        super().__init__(name, tool_opts, logger=logger, **kwargs)
+        if self.tool_dir:
+            self.tool_dir = self.tool_dir / self.name
+        pmcd_path = find_executable("pmcd")
+        if pmcd_path is None:
+            pmcd_path = self._pmcd_path_def
+        executable = os.access(pmcd_path, os.X_OK)
+        if executable:
```

Comment:
Seems it might be useful to encapsulate this; although I wonder whether `find_executable` already checks that the file actually is executable? Seems it ought to.

Reply:
I think it does.

Reply:
The logic here is that the second executable check is for the default path when find_executable fails to find it. It might still be redundant though, unsure.

Reply:
Let's not hard-code the paths to the PCP tools.  Instead, just make sure the PATH includes `/usr/libexec/pcp/bin`.  Then just use `find_executable`.